docs/source/llms/openai/autologging.rst

```diff
@@ -0,0 +1,184 @@
+MLflow OpenAI Autologging
+=========================
+
+The OpenAI flavor for MLflow supports autologging to ensure that experimentation, testing, and validation of your ideas can be captured dynamically without 
+having to wrap your code with logging boilerplate. 
+
+In this guide, we'll discuss some of the key features that are available in the autologging feature. 
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 1
+
+Quickstart
+----------
+To get started with MLflow's OpenAI autologging, you simply need to call :py:func:`mlflow.openai.autolog` at the beginning of your script or notebook. 
+Enabling autologging with no argument overrides will behave as the ``default`` configuration in the table in the next section. Overriding any of these settings 
+will allow you to log additional elements. 
+
+.. tip::
+    The only element that is **enabled by default** when autologging is activated is the recording of trace information. You can read more about MLflow tracing 
+    `here <../tracing/index.html>`_. 
+
+.. code-block:: python
+
+    import os
+    import openai
+    import mlflow
+
+    mlflow.openai.autolog()
+
+    openai_client = openai.OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
```

Comment:
```suggestion
    openai_client = openai.OpenAI()
```

I'd remove `api_key` because `OpenAI` automatically picks it up.

Reply:
Good idea. I'll put notes in instead.